hw/formal/tools/jaspergold/conn.tcl

```diff
@@ -5,12 +5,28 @@
 # clear previous settings
 clear -all
 
+# Save debug effort to escalate the following error messages
```

Comment:
just curious, would it make sense to put all these message filters into a separate TCL file that could then be shared among `conn.tcl` and `fpv.tcl`?

Reply:
The actual execution command is different, and coverage collection will be different as well. But I think you have a point, maybe I can have a base tcl file to set these setting.

Reply:
That is a good point! I will do that. Thanks Michael for the suggestion!